src/components/footer.tsx

```diff
@@ -0,0 +1,84 @@
+import { Todo } from '../types/Todo';
+import classNames from 'classnames';
+import * as todosService from '../api/todos';
+
+type Props = {
+  onFilter: (filter: string) => void;
+  onError: () => void;
+  onTodos: () => void;
+  todos: Todo[];
+  filter: string;
+};
+
+export const Footer: React.FC<Props> = ({
+  onFilter,
+  onError,
+  onTodos,
+  todos,
+  filter,
+}) => {
+  function clearCompleted() {
+    todos.map(todo => {
+      if (todo.completed) {
+        todosService
+          .deleteTodo(todo.id)
+          .catch(() => onError('Unable to delete todo'));
+      }
+    });
+    onTodos(todos.filter(todo => !todo.completed));
+  }
+
+  enum Status {
+    All = 'all',
+    Active = 'active',
+    Completed = 'completed',
+  }
```

Comment:
Why do you duplicate enum in other components? You need to move enum to the separate file and use this enum everywhere